holoviews/core/data/dask.py

```diff
@@ -229,12 +229,8 @@ def dframe(cls, columns, dimensions):
         return columns.data.compute()
 
     @classmethod
-    def length(cls, dataset):
-        """
-        Length of dask dataframe is unknown, always return 1
-        for performance, use shape to compute dataframe shape.
-        """
-        return 1
+    def nonzero(cls, dataset):
+        return True
 
```

Comment:
Matt Rocklin suggested using ``.head`` and checking the length of that, not any faster than my solution above though.